source/SpinalHDL/Getting Started/A simple example.rst

```diff
@@ -0,0 +1,131 @@
+.. _Simple example:
+
+A simple example
+================
+
+In ``MyTopLevel.scala`` is the following hardware description:
+
+.. code-block:: scala
+
+   case class MyTopLevel() extends Component {
+     val io = new Bundle {
+       val cond0 = in port Bool
+       val cond1 = in port Bool
```

Comment:
1) Bool alone should never be used excepted to specify a scala type.
Instead here, use Bool()

2) I'm not sure it is a good idea to already mix in the port syntax, as all the code base / legacy is without it.
Also in Bool()       in UInt(8 bits) is less typing.

Reply:
1. I wasn't aware of that, maybe it should be deprecated to warn users?
2. It is early, but I wanted to show in this PR how we could introduce this. Maybe we should do that after 1.8 is published?  
Also, after this `Getting started` section would be a tutorial with new examples / exercises which could adopt this new syntax, and still let a link to the full explanation with the legacy syntax which can be found in examples (until we modify it).
3. > Also in Bool() in UInt(8 bits) is less typing.
    * As mentioned in the PR for `port`, for beginners it is far simpler to explain "use `in port Type()`" than "you can use `in Type ()` but not for all types, so be careful"
    * There is written elsewhere in RTD that the scala style guide should be applied, and when **not** using port it would be `in Type ()` with an ugly space before the braces.  
    So there are two reasons why I do not want to use this non-universal poorly-formatted syntax, especially in a tutorial.

Reply:
As have I have moved this part outside the tutorial part, I'm reverting this to the old syntax.

Reply:
> I wasn't aware of that, maybe it should be deprecated to warn users?

Yes, it can be added. This will trigger a loooooooooot of user code base, that's for sure XD

> in Type ()

What is the ugly space about ? I mean, why having it ?

About universal syntax, it was in(Type()) so far.

Reply:
I'll open an issue for the deprecation warning

> What is the ugly space about ? I mean, why having it ?

Because it is infix operator `Type` syntax, so it is as when you write `a + b`, there are spaces before and after `+`.

It seems you completely forgot discussion about it https://github.com/SpinalHDL/SpinalHDL/pull/935#issuecomment-1292357220 even if you took part in it https://github.com/SpinalHDL/SpinalHDL/pull/935#issuecomment-1293363216 :sweat_smile: